core/java/iesi-rest-without-microservices/src/test/java/io/metadew/iesi/server/rest/script/ScriptControllerTest.java

```diff
@@ -707,6 +706,17 @@ void getByNameAndVersion404() throws Exception {
                 .andExpect(status().isNotFound());
     }
 
+    @Test
+    void getByNameAndVersionFile() throws Exception {
```

Comment:
create a negative test: try to download a script name - script version that not exists